vertx-web/src/main/java/io/vertx/ext/web/handler/sse/SSEConnection.java

```diff
@@ -0,0 +1,68 @@
+package io.vertx.ext.web.handler.sse;
+
+import io.vertx.codegen.annotations.Fluent;
+import io.vertx.codegen.annotations.GenIgnore;
+import io.vertx.codegen.annotations.VertxGen;
+import io.vertx.core.http.HttpServerRequest;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.ext.web.handler.sse.impl.SSEConnectionImpl;
+
+import java.util.List;
+
+@VertxGen
+public interface SSEConnection {
+
+	static SSEConnection create(RoutingContext context) {
+		return new SSEConnectionImpl(context);
+	}
+
+	@Fluent
+  SSEConnection forward(String address);
+
+	@Fluent
+  SSEConnection forward(List<String> addresses);
+
+	@Fluent
+  SSEConnection reject(int code);
```

Comment:
I'm not sure about these reject `methods`. What are the use cases? I can only think about authentication, which should be managed by other Vert.x Web handlers.